1-exercises/B-object-arrays/exercise-1.js

```diff
@@ -13,8 +13,8 @@ let people = [
   { name: "Elise", age: 35 },
 ];
 
-let peopleNames =
-  /* todo */
+let peopleNames = people.map((item) => Object.values(item));
+/* todo */
 
-  // DO NOT EDIT BELOW THIS LINE
-  console.log(`Expected output: Bob,John,Elise. Actual output: ${peopleNames}`);
+// DO NOT EDIT BELOW THIS LINE
+console.log(`Expected output: Bob,John,Elise. Actual output: ${peopleNames}`);
```

Comment:
In my coursework, there's a 'exercise-2.js' in 'B-ojject-arrays'.

Reply:
Correct @howard-ss  What is the problem :)
Here is the link to that [B-object-arrays/exercise-2.js](https://github.com/CodeYourFuture/JavaScript-Core-2-Coursework-Week2/blob/main/1-exercises/B-object-arrays/exercise-2.js)